.github/workflows/main.yml

```diff
@@ -34,6 +34,7 @@ jobs:
   
   build-and-test:
     env:
+      CC: gcc-8
```

Comment:
Change in master needed for this build to pass that hasn't been merged into the base branch yet.